.ci/e2e_integration_test/start-e2e.ps1

```diff
@@ -89,8 +89,9 @@ Write-Host "Preparing E2E integration tests..." -ForegroundColor Green
 Write-Host "-----------------------------------------------------------------------------`n" -ForegroundColor Green
 python -m pip install -U pip
 python -m pip install -U -e .[dev]
-python setup.py build
-python setup.py extension
+cd tests
+python -m invoke -c test_setup build-protos
```

Comment:
In PR description, it says test_setup build_protos. Is build-protos the correct way?

Reply:
Updated the description. Its correct in the code. build-protos